reporters_db/data/reporters.json

```diff
@@ -27302,7 +27702,7 @@
                 "U.S. CONST. AMEND. 9"
             ],
             "mlz_jurisdiction": [],
-            "name": "U.S. Constitution",
+            "name": "Constitution of The United States of America, USCS",
```

Comment:
This seems odd... what is the USCS stand for?  Also probably meant for the laws.json

Reply:
I think it stands for "United States Code Service"